src/typescript/decorators.d.ts

```diff
@@ -0,0 +1,54 @@
+/// <reference path="../../node_modules/reflect-metadata/Reflect.d.ts" />
```

Comment:
Maybe? Typescript decorators have diverged from the current proposals for decorators, so "spec" is probably a loose term at this point.